src/QMCHamiltonians/QMCHamiltonian.h

```diff
@@ -266,6 +266,10 @@ class QMCHamiltonian
       std::vector<RealType>& dhpsioverpsi,
       bool compute_deriv);
 
+  Return_t evaluateIonDerivs(ParticleSet& P, ParticleSet& ions, TrialWaveFunction& psi,
```

Comment:
Please add documentation for this function

Reply:
Done